ops/internal/manage/depsets.go

```diff
@@ -0,0 +1,230 @@
+package manage
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-supervisor/supervisor/backend/depset"
+	"github.com/ethereum-optimism/superchain-registry/ops/internal/config"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+var (
+	errInvalidActivationTime = errors.New("activation_time is before interop_time")
+	errDepsetLengths         = errors.New("inconsistent depset lengths")
+	errInconsistentDepsets   = errors.New("inconsistent depset values")
+	errMissingAddress        = errors.New("missing address")
+)
+
+type DepsetChecker struct {
+	lgr             log.Logger
+	addrs           config.AddressesJSON
+	diskChainCfgs   map[uint64]DiskChainConfig
+	processedChains map[uint64]bool
```

Comment:
Any reason not to just use `eth.ChainID` as the key and for all the chain ID types? Would save checking it is a uint64 and a bunch of conversions later on.  eth.ChainID is safe to use as a map key.

Reply:
The issue is that in the chain config tomls chainId is stored as a `uint64` currently ([ref](https://github.com/ethereum-optimism/superchain-registry/blob/main/ops/internal/config/chain.go#L72)). So I think we'll have to do roughly the same amount of conversions between types either way. It just moves the conversion if we change the type you pointed out to be `eth.ChainId`. Unless we change the chain config toml chainId type

Reply:
That counts as reason enough to me. :) Its not a big deal.